content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a certificate in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. The following example downloads and installs the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin assuming the environment variables are set for the plugin version and install path for your development computer.
+
+{{% alert title="Warning" color="primary" %}}
+The following example does not work unless you set the environment variables, and other plugins may follow a different process for downloading. You also need to know the specific location `{NOTATION_LIBEXEC}/plugins/` on your development computer. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/concepts/directory-structure" >}}).
+{{% /alert %}}
+
+```console
+curl -Lo ${tar_file} "https://github.com/Azure/notation-azure-kv/releases/download/v${version}/${tar_file}"
+mkdir -p ${install_path}
+tar xvzf ${tar_file} -C ${install_path} notation-azure-kv
+```
+
+To confirm you plugin is installed, run `notation plugin list`. For example:
+
+```console
+notation plugin list
+```
+
+## Uninstall a plugin
+
+To remove a plugin, delete the entire `{NOTATION_LIBEXEC}/plugins/{plugin-name}/` directory.
+
+To confirm you plugin is uninstalled, run `notation plugin list`. For example:
+
+```console
+notation plugin list
+```
+
+## Plugin naming structure
+
+The `notation` CLI strictly follows the [Notary Project specification for plugins](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#plugin-mechanism). This includes the naming structure for the plugin binary and the plugin directory. All plugins must be named `notation-{plugin-name}` and placed in the `{NOTATION_LIBEXEC}/plugins/{plugin-name}/` directory. For example, the full path to the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin is `{NOTATION_LIBEXEC}/plugins/azure-kv/notation-azure-kv`.
+
+If the plugin name or plugin directory does not match the naming structure, `notation` will not recognize the plugin.
+
+## Security best practices for plugins
```

Comment:
should there also be a section calling out risks around plugins (and thus why it's important to make sure that plugins come from a trusted source)? ex:

- plugins are invoked as binaries, so there's not a lot of visibility into specifically what they're doing
- notation passes signature/payload/key information to the plugin which may be sensitive